core/kazoo_ast/src/fs_prop_usage.erl

```diff
@@ -0,0 +1,648 @@
+-module(fs_prop_usage).
+
+%% module for parsing ecallmgr modules, looking for Props from FreeSWITCH
+
+-export([process/0, process/1
+        ,to_header_file/0
+        ]).
+
+-include_lib("kazoo_ast/include/kz_ast.hrl").
+-include_lib("kazoo/src/kz_json.hrl").
+
+-define(DEBUG(_Fmt, _Args), 'ok').
+%% -define(DEBUG(Fmt, Args), io:format([$~, $p, $  | Fmt], [?LINE | Args])).
+
+to_header_file() ->
+    Usage = process(),
+    write_usage_to_header(Usage).
+
+write_usage_to_header(Usage) ->
+    {'ok', IO} = file:open(event_filter_filename()
+                          ,['write'
+                           ,'delayed_write'
+                           ]),
+    write_usage_to_header(Usage, IO).
+
+write_usage_to_header(Usage, IO) ->
+    EventFilters = lists:foldl(fun write_mod_usage/2, sets:new(), Usage),
+    [First|Sorted] = lists:usort(sets:to_list(EventFilters)),
```

Comment:
meh, just being __really__ sure the list is sorted and only contains the unique items.